README.md

```diff
@@ -28,21 +25,32 @@ If this is your first time running a React Native project on this machine, follo
 
 ### Keys/Secrets
 
-You will need to get your own Mapbox API Key. Globally find `[MAPBOX_API_KEY]` and replace with your key.
+You will need to get your own Mapbox API Key & Google Maps API Key. Place them in **`env.json`** before you start.
```

Comment:
IIRC, the Google Maps API key stuff can be removed from the project. We're relying solely on Mapbox's Distance API. (We had started with Google's. It was better (included traffic in distance calculations), but we removed that because of the expenses and low API call limits and opted for Mapbox's instead. So removing the Google API stuff altogether would be an improvement.